frontend/src/compliance_reporting/components/ComplianceReportingTable.js

```diff
@@ -123,18 +78,16 @@ class ComplianceReportingTable extends Component {
       id: 'supplemental-status',
       minWidth: 75
     }, {
-      accessor: item => (item.updateTimestamp ? moment(item.updateTimestamp).format('YYYY-MM-DD') : '-'),
+      accessor: item => (item.sortDate ? item.sortDate : null),
```

Comment:
This will break the filtering. Ideally, the code for the new format is here in the accessor instead of Cell

Reply:
Resolved in a way that filter and sort both work